api.example.yml

```diff
@@ -60,8 +60,8 @@ web:
       placeholder: "Slack #channel or @user"
     - type: telegram
       label: Telegram
-      placeholder: "#channel, @username or group"
-      help: "### To make things work you should:\n### In personal chat:\n - start conversation with bot [@YourMoiraBot](https://t.me/YourMoiraBot);\n - execute command `/start`;\n - type your login in above field as `@login`.\n\n### In group chat:\n - invite bot [@YourMoiraBot](https://t.me/YourMoiraBot) into chat;\n - execute command `/start@YourMoiraBot`;\n - bot will send you chat name, you should type it without extra characters in above field.\n\n### In channel:\n - add bot [@YourMoiraBot](https://t.me/YourMoiraBot) into channel;\n - promote bot as channel administrator;\n - type channel name in above field as `#channel`.\n"
+      placeholder: "#public_channel, %private_channel, @username or group"
+      help: "### To make things work you should:\n### In personal chat:\n - start conversation with bot [@YourMoiraBot](https://t.me/YourMoiraBot);\n - execute command `/start`;\n - type your login in above field as `@login`.\n\n### In group chat:\n - invite bot [@YourMoiraBot](https://t.me/YourMoiraBot) into chat;\n - execute command `/start@YourMoiraBot`;\n - bot will send you chat name, you should type it without extra characters in above field.\n\n### In public channel:\n - add bot [@YourMoiraBot](https://t.me/YourMoiraBot) into channel;\n - promote bot as channel administrator;\n - type channel name in above field as `#channel`.\n\n### In private channel:\n - add bot [@YourMoiraBot](https://t.me/YourMoiraBot) into the channel;\n - promote bot as channel administrator;\n - open your private channel on the [web](https://web.telegram.org/#/im);\n - get channel id from URL (e.g., `https://web.telegram.org/#/im?p=c1494975744_17340166617136722341`) between `c` and `_`;\n - type channel id in the above field as `%1494975744`.\n"
```

Comment:
Let's divide this huge and ling string to multi-line string that is supported by yaml
https://yaml-multiline.info

Reply:
done